roles/install-flannel/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+- name: Download flannel manifest
+  get_url:
+    url: "https://raw.githubusercontent.com/flannel-io/flannel/v0.26.1/Documentation/kube-flannel.yml"
```

Comment:
try to control this version from the group vars so that you can update later in the config

Reply:
Added flannel_version variable to control this from group vars